lib/report_portal/settings.rb

```diff
@@ -46,18 +46,28 @@ def initialize
         'use_standard_logger' => false,
         'launch_id' => false,
         'file_with_launch_id' => false,
+        'launch_uuid'=>true #used when multiple cucumber processes executed 
```

Comment:
Could you please explain the use case for `launch_uuid` in more detail?

Reply:
if you strat bunch of cucumber processes with shared lock folder, the script that triggers processes can simply generate UUID key and pass it to cucumber process that will then create lunch and share lunch_id using lock file

Reply:
I didn't understand that, unfortunately.
Is that for running several instances of parallel_tests? i.e. run parallel_tests, then before they finish run parallel_tests again. Or for something else?

Reply:
it's to run multiple cucumber processes(in different processes or even hosts )